### PR TITLE
Catch API errors that are generated in the end-to-end tests

### PR DESCRIPTION
**Pull request by teneightfive**
### What changed

Catch API errors that occur when running end-to-end tests and log them to Sentry.

### Why did it change

Currently in some scenarios the API may return a `422` when we
create a dynamic fixture.

It's hard to debug to see what extactly went wrong so this change
adds the ability to catch those errors and log them to Sentry so that
they can be investigated.

## Screenshots

<kbd>image</kbd>

<kbd>image</kbd>

<kbd>image</kbd>

### CODE REVIEW

File: test/e2e/_helpers.js [hunk 1]
```diff
@@ -14,8 +15,30 @@ import personService from '../../common/services/person'
...
+}
+
+function errorHandler(body) {
+  return err => {
+    Sentry.withScope(scope => {
+      err.errors.forEach((apiErr, idx) => {
+        scope.setContext(`error_${idx}`, apiErr)
+      })
+      scope.setContext('body', body)
+      Sentry.captureException(err)
```
Comment (teneightfive) [inline test/e2e/_helpers.js:35]: I'm not sure what level we want to capture these as: `error`, `warning`, `info`. The default is `error`.

They will be filtered out already by the `environment` value and won't create noise in the main alerts channel so maybe `error` is okay.

File: test/e2e/_helpers.js [hunk 2]
```diff
@@ -14,8 +15,30 @@ import personService from '../../common/services/person'
...
+  })
+}
+
+function errorHandler(body) {
+  return err => {
+    Sentry.withScope(scope => {
+      err.errors.forEach((apiErr, idx) => {
+        scope.setContext(`error_${idx}`, apiErr)
+      })
+      scope.setContext('body', body)
```
Comment (teneightfive) [inline test/e2e/_helpers.js:34]: This does send the body data to Sentry but in our cases it is fake data generated using faker so won't contain anything sensitive/PII.

File: test/e2e/_helpers.js (hunk 1 above)
Comment (solidgoldpig) [reply test/e2e/_helpers.js:35]: Sounds reasonable

**Review by solidgoldpig — APPROVED** [review]

File: test/e2e/_helpers.js (hunk 1 above)
Comment (merlinc) [reply test/e2e/_helpers.js:53]: Yeah, let's see how noisy `error` is, and work from there.

File: test/e2e/move.update.sch.test.js [hunk 3]
```diff
@@ -47,10 +46,6 @@ test.meta('hasDocument', 'true')(
   }
 )
 
-test('User should be able to cancel move', async () => {
```
Comment (merlinc) [inline test/e2e/move.update.sch.test.js:50]: These were added in by me, as this functionality was specified in the original story - but if they're unduly slowing down the tests, they're probably one of the safest tests to remoe.

**Review by merlinc — APPROVED** [review]

**Pull request merged**